platforms/nuget/build-jinja.py

```diff
@@ -1,32 +1,78 @@
-import os
+import os, argparse
+from pathlib import Path
+from shutil import copyfile
 from jinja2 import Environment, FileSystemLoader
 PATH = os.path.dirname(os.path.abspath(__file__))
 TEMPLATE_ENVIRONMENT = Environment(
     autoescape=False,
     loader=FileSystemLoader(os.path.join(PATH, 'templates')),
     trim_blocks=False)
 
+VERSION_PATH = str(Path(PATH).parents[1]) + "/modules/core/include/opencv2/core/version.hpp"
+version_file = Path(VERSION_PATH).read_text()
+
+version_major = int(version_file.split("CV_VERSION_MAJOR")[1].split("\n")[0])
```

Comment:
Why cast to int?

Reply:
The cast was done in order to avoid spaces in the string. It has now been replaced with a better .replace method for strings in order to avoid the unnecessary cast.